stl/inc/experimental/generator

```diff
@@ -33,18 +32,18 @@ _STL_DISABLE_CLANG_WARNINGS
 _STD_BEGIN
 
 namespace experimental {
-
-    template <typename _Ty, typename _Alloc = allocator<char>>
+    // NOTE WELL: _CPPUNWIND currently affects the ABI of generator. Linking together /EHs
+    // translation units and /EHs- translation units that use the same specialization of generator
+    // violates the One Definition Rule. This could result in fantastic failures or obscure bugs.

```

Comment:
This comment should avoid `/EHs-` which isn't supported:
```suggestion
    // NOTE WELL: _CPPUNWIND currently affects the ABI of generator.
```
(I don't think we need to mention linking /kernel and non-/kernel TUs which would be a bizarre thing to do regardless.)